sampledkd/training/entrypoint_utils.py

```
@@ -0,0 +1,432 @@
+        return teacher, tok, torch.device(f"cuda:{one}")
+    except Exception as e:
+        print(f"[teacher] 8-bit failed: {e}", flush=True)
+
+    # ===== 1.5) Force 8-bit on a single GPU (no CPU offload) =====
+    try:
+        teacher, tok, teacher_inputs_device = load_teacher_8bit_strict(
+            model_name="Qwen/Qwen3-8B",
+            prefer_gpus=[0,1],     # local indices on the 2080 Ti box
+            student_gpu=1,         # keep GPU 1 free for the student
+        )
+    except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)